src/lib/reasoners/satml_frontend.ml

```diff
@@ -597,7 +616,7 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
             let acc =
               List.fold_left (fun (mz,lz) f ->
                   match atoms_from_sat_branch f with
-                  | None -> raise Exit
+                  | None -> assert false
```

Comment:
This code is within a `try .. with Exit -> None`, can you explain this change?

Reply:
A mistake, I'll just revert the commit